### ISSUE TITLE
FEATURE: Translate history

### ISSUE THREAD

**Issue by mrkeuz** [artemave/translate_onhover#86]
Hi, thanks for great extension, it is simple and minimalistic. 🚀

- It would be good to save and ability export translation history.

Use case: 
I learn language with `Anki` cards and want to add collect all unknown words. Translator context is the best way where you can get data about new words for you. I.e. I've already collect such history from browser history (from Google Translate URLs). It will be cool to do same with your extension.

**Comment by artemave** [comment]
I agree, that would be a useful feature. However, I am not actively developing transover at the moment, so don't expect it any time soon.

**Comment by mrkeuz** [comment]
Seems considerable duplicate #111 and #113, so closed.
